Corona-Warn-App/src/main/res/layout/tracing_details_find_details_in_journal.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="state"
+            type="de.rki.coronawarnapp.tracing.states.TracingFailed" />
+    </data>
```

Comment:
can we please avoid using DataBinding here ?

Reply:
certainly. i don't think we need it